src/lighteval/few_shot_manager.py

```diff
@@ -220,7 +219,7 @@ def get_examples(
         return instruction + labeled_examples + example
 
     def create_multi_turn_contexts(
```

Comment:
I will create a follow-up PR for multi-turn contexts to work with `ChatCompletionInputMessage` instead of str in `FewshotManager` and `BaseModel` .